admin/adminHandler.go

```diff
@@ -194,3 +182,47 @@ func (contextHandler *AdminHandlerContext) HandleUpdateAdmin(w http.ResponseWrit
 
 	fmt.Fprintf(w, "admin updated successfully")
 }
+
+func (contextHandler *AdminHandlerContext) HandleAdminCreateHealthInsurence(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		http.Error(w, "Method is not supported.", http.StatusMethodNotAllowed)
+		return
+	}
+
+	var healthinsurance models.Healthinsurance
+
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&healthinsurance); err != nil {
+		http.Error(w, fmt.Sprintf("Error decoding JSON: %s", err), http.StatusBadRequest)
+		return
+	}
+
+	err := healthinsurance.Insert(context.Background(), contextHandler.Db, boil.Infer())
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Error inserting user: %s", err), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusCreated)
+	fmt.Fprintf(w, "HealthInsurece created successfully")
```

Comment:
Typo here, @ `HealthInsurece` (skipped an `n`)
Actually, "insurance" is spelled with an `a`, but changing this will affect too many parts, so you can just add the missing `n` for now